src/apps/bridge/app_config.h

```diff
@@ -19,12 +19,14 @@ constexpr const char *SAMPLES_PER_REPORT = "samplesPerReport";
 constexpr const char *CURRENT_READING_PERIOD_MS = "currentReadingPeriodMs";
 constexpr const char *SOFT_READING_PERIOD_MS = "softReadingPeriodMs";
 constexpr const char *RBR_CODA_READING_PERIOD_MS = "rbrCodaReadingPeriodMs";
+constexpr const char *kTicksSamplingEnabled = "ticksSamplingEnabled";
```

Comment:
Haha, ok, so consistency is the most important thing here. Per our recent conversation we could either do constants `LIKE_THIS` or `kLikeThis`. You and I both searched and found no super compelling reason to do one over the other.

I have a slight preference to avoid SHOUTING, so I would support this change to the `k` prefix. However, consistency is queen — if we change the style of this one, we need to change at least all the constants in this file to the new style.

Reply:
I'll change to use to use CAPS for the moment - if we decide to do a cleanliness thing later we can change it then.